src/main/resources/hudson/plugins/s3/S3BucketPublisher/global.jelly

```diff
@@ -9,7 +9,7 @@
             <f:textbox name="name" value="${profile.name}" />
           </f:entry>
           <f:entry title="Use IAM Role" help="/plugin/s3/help-role.html">
-            <f:checkbox name="useRole" value="${profile.useRole}" />
+            <f:checkbox name="useRole" value="${profile.useRole}" checked="${profile.useRole}"/>
```

Comment:
Sorry for the silence here. I just tested and here is my suggestion. `value` is not required here. In 24hr I will either merge the PR with it or do the change myself and merge the PR and release the plugin.

```suggestion
            <f:checkbox name="useRole" checked="${profile.useRole}"/>
```